Formula/a/apitrace.rb

```diff
@@ -0,0 +1,133 @@
+class Apitrace < Formula
+  desc "Tools for tracing OpenGL, Direct3D, and other graphics APIs"
+  homepage "https://apitrace.github.io/"
+  url "https://github.com/apitrace/apitrace/archive/00706d1a395ab6570ab88136fffb3028a3ded3bd.zip"
+  version "11.1+dev.0"
+  sha256 "0b51d3d6e9db9b42fc47c3ffad5b9ae68054a2a0e518f4c8c287316842c04757"
+  license all_of: ["MIT", "BSD-3-Clause", "libpng-2.0", "Zlib"]
+
+  # Known problems:
+  # - libbacktrace only meant to be used for ELF support / only relevant for Linux
```

Comment:
This will be relevant for linux, which I haven't tested at all (I know this is part of the cookbook; just didn't get to it yet and not sure how to test yet)